README.md

```diff
@@ -14,8 +14,59 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the format of environment variables. They 
+may be a bit different between packages, but the most common are:
+    BRANCH
+    VERSION
+    RELEASE
+
+So in order to build an specific branch / version, this command can be used from
+the folder of the package we want to build:
+
+    make BRANCH=qa/1.x VERSION=1.7.0 RELEASE=rc5
+
+Have in mind that the Makefiles are a bit recursive, they will invoke docker, 
+mount the current folder, and run the deb-build target.
+
+#### Development
+
+Some makefiles have a "dev" target, that give you a shell inside of the docker 
+used to build packages. When you are inside the container, the command needed to
+build packages is:
+
+    make deb-build
+
+### Repositories management
+
+This repo generates Archivematica packages, but also packages that need to be 
+installed in order for Archivematica to run. They are placed in the 
+ubuntu-extras or centos-extras repos.
+
+In order to add a package to a repo, once its built and uploaded to a temporary folder at https://packages.archivematica.org, the steps are:
+
+For centos packages:
+
+  - Copy the rpm into centos-extras repo
+  - Run ``createrepo``
+  - Run '``gpg --detach-sign --armor repodata/repomd.xml`` to sign the 
+repository contents
+
+For debian packages:
+
+  - Upload the package to packages.archivematica.org
+  - Go to the repository folder ( mnt/st-sites-pub/packages.archivematica.org/1.7.x/ubuntu-externals )
+  - Add the packages with
+
+     ``reprepro includedeb trusty /path/to/packages/*.deb``
+     ``reprepro includedsc trusty /path/to/packages/*.deb``
```

Comment:
These are showing up on a single line. I think you could remove the backticks and just indent lines 62/63 two more spaces and it would show up as a code block on two lines.

Reply:
After a bit of a fight, done